extensions/default/src/getCustomizationModule.tsx

```diff
@@ -136,6 +141,20 @@ export default function getCustomizationModule() {
             return clonedObject;
           },
         },
+
+        {
+          id: 'ohif.cloudServerConfigurationComponent',
+          component: CloudServerConfigurationComponent.bind(null, {
+            servicesManager,
+            extensionManager,
+          }),
+        },
```

Comment:
how is this different than the below one which is `ohif.cloudServerAPI.google`. Is this a more general one?

Reply:
This one is the GUI component.

Reply:
I think i understand now that I read the code, but comments above each of these is appreciated by reader